app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split(" ")
```

Comment:
Consider using more descriptive variable names for clarity. For example, instead of `files`, you could use `command_parts` to indicate that this variable holds parts of the command string. This aligns with the checklist's recommendation to use descriptive and correct variable names.